src/components/TracePage/SpanGraph/index.css

```diff
@@ -20,19 +20,23 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 */
 
-.span-graph--tick {
-  stroke: #aaa;
-  stroke-width: 1px;
+.SpanGraph--zlayer {
+  position: relative;
+  z-index: 1;
 }
 
-.span-graph--tick-header {
+.SpanGraph--graph {
+  border: 1px solid #999;
+  overflow: visible !important;
```

Comment:
Is there a particular reason we need to use `!important` here? 
Might be better to inline it on the element to be more explicit if it is a functional requirement

Reply:
Yes, we need it to overcome this from semantic UI:

```css
svg:not(:root) {
    overflow: hidden;
}
```

Will add a comment.

Thanks.